cou/steps/backup.py

```diff
@@ -44,9 +44,12 @@ async def backup(model: Model) -> Path:
     await model.run_on_unit(unit_name, f"chmod o+rx {basedir}")
 
     local_file = COU_DATA / os.path.basename(remote_file)
-    logger.info("SCP from  mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
+    logger.info("SCP from mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
     await model.scp_from_unit(unit_name, remote_file, str(local_file))
 
+    progress_indicator.info(f"Backed up MySQL to {local_file}")
+    progress_indicator.start()
```

Comment:
(question) I am not super familiar with `halo`, is this valid? should we `start` before `info`? do we need `start` and without `stop`

Reply:
`info` will stop the progress_indicator.

All the steps have `progress_indicator.start` at  https://github.com/canonical/charmed-openstack-upgrader/blob/0d3968d8998fb05df7eb3ab8369eb4ef44116a4d/cou/steps/execute.py#L120 .

Reply:
thanks

Reply:
I think we should use `progress_indicator.start(f"Backed up MySQL to {local_file}")`, what is the benefit of using info and then start? Should we also do stop here `progress_indicator.succeed`?

Reply:
I think the idea is to persist the message on the screen. It's not starting a process, it's logging a message. Probably using `print()` is fine like in other places in COU's codebase.

Or perhaps we could add the local path to the progress indicator for the backup step, so we need to touch the progres indicator from within the backup function at all? I'm thinking about it now, and that would actually make more sense, because the backup() function shouldn't be managing a progress indicator that is defined and managed outside the function - it makes it less portable.

Reply:
If you want to persist it, you can use `progress_indicator.start(f"Backed up MySQL to {local_file}")` and then `progress_indicator.succeed()` without message. Like that, the message will stay there, at least that's how I remember how it works.